File: 문자찾기.java
import java.util.Scanner;

class Main {
  public int solution(String str, char t) {
  	int answer = 0;
    str = str.toUpperCase();
    t = Character.toUpperCase(t);
    for(int i = 0; i < str.length(); i++) {
    	if(str.charAt(i) == t) answer++;
    }
    
    return answer;
  }
  
  public static void main(String[] args){
    Main T = new Main();
    Scanner in=new Scanner(System.in);
    String str = in.next();
    char c = in.next().charAt(0);
    
    System.out.println(T.solution(str, c));
  }
}

File: 문자찾기.kt
import java.util.Scanner

class Main {
    fun solution(str: String, t: char) : Int {
        var answer = 0;
        val upperCasestr = str.toUpperCase()
        val upperCaset = t.toUpperCase()
        for(i in 0 until upperCasestr.length) {
            if(upperCasestr[i] == upperCaset) answer++
        }

        return answer
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val T = Main()
            val scanner = Scanner(System.in)
            val str = scanner.next();
            val c = scanner.next()[0]
            println(T.solution(str, c))
        }
    }

}